frontend/web/src/app/components/sidemenu/sidemenu.component.html

```diff
@@ -14,30 +14,30 @@
       <li>
         <a routerLink="orders" class="nav-link" href="#" routerLinkActive="active">Orders</a>
       </li>
-      <li class="dropdown">
-        <a href="#" class="dropdown-toggle" data-toggle="dropdown">Parts
-          <span class="caret"></span>
+      <li>
+        <a href="#parts" class="collapsed" data-toggle="collapse">Parts
+          <i class="glyphicon glyphicon-menu-down"></i>
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this <i> tag by <em>. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)